service/build.gradle

```diff
@@ -6,6 +6,7 @@ plugins {
   id 'org.springframework.boot' version '2.1.9.RELEASE'
   id 'com.github.ben-manes.versions' version '0.25.0'
   id 'org.sonarqube' version '2.8'
+  id 'project-report'
```

Comment:
I am not convinced we need that. There are already command line tasks that provides same information just printed  to the terminal. For example to see dependencies tree `dependencies` task is already available.

Reply:
yes you are right, adding this was just a nice to have as we can have the html 😄